queue_job/controllers/main.py

```diff
@@ -73,7 +73,7 @@ def old_runjob(self, db, job_uuid, **kw):
 
     @http.route('/queue_job/runjob', type='http', auth='none')
     def runjob(self, db, job_uuid, **kw):
-
+        http.request.session.db = db
```

Comment:
@adrienpeiffer Why is this needed now and not in 9.0?

Perhaps it's the job runner that should `GET /queue_job/runjob?db=<dbname>` ?